File: backend/Dtos/Account/UserDto.cs
namespace backend.Dtos.Account
{
    public class UserDto
    {
        public string Id { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }

        public string Fullname { get; set; }

        public DateTime DateOfBirth { get; set; }
        public string PhoneNumber { get; set; }

        public string Gender { get; set; }

        public string City { get; set; }

        public string ImagePath { get; set; }
        
        public bool IsDeleted { get; set; }
        public DateTime DeletedAt { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: frontend/src/app/interfaces/User/UserDto.ts
export interface UserDto {
  id:string;
  username: string;
  email: string;
  fullname: string;
  dateOfBirth: Date;
  gender: string;
  phoneNumber:string;
  city: string;
  imagePath: string;
  isDeleted?: boolean;
  deletedAt?:Date;
}
